app/jobs/in_person/enrollments_ready_for_status_check_job.rb

```diff
@@ -0,0 +1,107 @@
+module InPerson
+  # This job checks a queue regularly to determine whether USPS has notitied us
+  # about whether an in-person enrollment is ready to have its status checked. If
+  # the enrollment is ready, then this job updates a flag on the enrollment so that it
+  # will be checked earlier than other enrollments.
+  class EnrollmentsReadyForStatusCheckJob < ApplicationJob
+    queue_as :low
+
+    def perform(_now)
+      return true unless IdentityConfig.store.in_person_proofing_enabled &&
+                         IdentityConfig.store.in_person_enrollments_ready_job_enabled
```

Comment:
This is just style, I really dislike `unless`, *especially* with a compound statement, I think its always clearer as an `if`

```suggestion
      if !IdentityConfig.store.in_person_proofing_enabled ||
        !IdentityConfig.store.in_person_enrollments_ready_job_enabled
        return true
      end
```

Reply:
I disagree; `unless` provides a larger visual indication of the conditional behavior than the oft-missed exclamation mark.

Reply:
maybe the conditions could be pulled into well-named methods, ie:

```ruby
def in_person_proofing_not_enabled
  !IdentityConfig.store.in_person_proofing_enabled
end
```

then it could look like this, which imo is a little easier to parse/reason about

```ruby
if in_person_proofing_not_enabled || in_person_enrollements_ready_job_not_enabled
  return true
end
```

Reply:
`blank?` will return `true` if a variable is `nil` or `false`
```ruby
      return true if IdentityConfig.store.in_person_proofing_enabled.blank? &&
                     IdentityConfig.store.in_person_enrollments_ready_job_enabled.blank?
```

Reply:
@tomas-nava That sounds like a reasonable compromise.

@Sgtpluck I think locality is preferable here.